x-pack/plugins/apm/server/lib/alerts/register_error_count_alert_type.ts

```diff
@@ -116,18 +95,19 @@ export function registerErrorRateAlertType({
         ESSearchRequest
       > = await services.callCluster('search', searchParams);
 
-      const value = response.hits.total.value;
+      const errorCount = response.hits.total.value;
 
-      if (value && value > alertParams.threshold) {
+      if (errorCount > alertParams.threshold) {
         const alertInstance = services.alertInstanceFactory(
-          AlertType.ErrorRate
+          AlertType.ErrorCount
         );
         alertInstance.scheduleActions(alertTypeConfig.defaultActionGroupId, {
           serviceName: alertParams.serviceName,
+          environment: alertParams.environment,
+          threshold: alertParams.threshold,
+          triggerValue: errorCount,
```

Comment:
All threshold alerts now expose the threshold and the value that triggered the alert. They are consistently named `threshold` and `triggerValue` to make it easier to reuse templates between alerts.